### PR TITLE
cmd: skeleton for segment-reaper detect command

### PR DESCRIPTION
**Pull request by mniewrzal**
What: Initial change for `segment-reaper detect` command. Only command skeleton.

Why: https://storjlabs.atlassian.net/browse/V3-3156

Please describe the tests:
 - Test 1: none
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: cmd/segment-reaper/detect.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+func cmdDetect(cmd *cobra.Command, args []string) (err error) {
+	log := zap.L()
+	db, err := metainfo.NewStore(log.Named("pointerdb"), detectCfg.DatabaseURL)
+	if err != nil {
+		return errs.New("error connecting database: %+v", err)
+	}
+	defer func() {
+		err = errs.Combine(err, db.Close())
+	}()
+	return err
```
Comment (ifraixedes) [inline cmd/segment-reaper/detect.go]: It's clearer to return `nil` despite `err` is nil at this point

File: cmd/segment-reaper/detect.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+var (
+	detectCmd = &cobra.Command{
+		Use:   "detect",
+		Short: "Detects zombie segments in DB",
+		Args:  cobra.OnlyValidArgs,
+		RunE:  cmdDetect,
+	}
+
+	detectCfg struct {
+		DatabaseURL string `help:"the database connection string to use" default:"postgres://"`
```
Comment (ifraixedes) [inline cmd/segment-reaper/detect.go]: Can we have this flag global to the 2 commands that this tool will have?

File: cmd/segment-reaper/detect.go (hunk 1 above)
Comment (mniewrzal) [reply cmd/segment-reaper/detect.go]: done

File: cmd/segment-reaper/detect.go (hunk 2 above)
Comment (mniewrzal) [reply cmd/segment-reaper/detect.go]: I prefer to leave it like that. List of global flags is already full of debug and other flags. Its also cleaner to have this flag with other flags related to this command.

**Pull request merged**